sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Release History
 
-## 5.3.0-beta.2 (Unreleased)
+## 5.3.0-beta.2 (2020-09-29)
+
+### Changes
+
+#### New Features
+
+- The `EventData` representation has been extended with the ability to treat the `Body` as `BinaryData`, providing support for a variety of data transformations and `ObjectSerializer` integration.
+
+- `EventData` has been integrated with the new Schema Registry service, via use of the `ObjectSerializer` with `BinaryData`.
```

Comment:
I'd go with something more like:
> Accessing `Body` as `BinaryData` allows the ability to provide serialization logic when sending or receiving events. Any type that derives of `ObjectSerializer`, such as `JsonObjectSerializer` and `SchemaRegistryAvroObjectSerializer`, can utilize this workflow. The aforementioned Avro serializer utilizes the Azure Schema Registry service.

Reply:
I like that.   Thanks!

Reply:
Gave it a bit of wordsmithing, but kept the theme.